controller/lib/core/sensors.cpp

```diff
@@ -101,34 +90,48 @@ void sensors_init() {
   // memory and provide operators with a way to shut off the device's blowers,
   // open any necessary valves, and recalibrate.
   Hal.delay(20);
-  sensorZeroVals[static_cast<int>(PressureSensors::PATIENT_PIN)] =
-      get_raw_sensor_zero_reading(PressureSensors::PATIENT_PIN);
-  sensorZeroVals[static_cast<int>(PressureSensors::INHALATION_PIN)] =
-      get_raw_sensor_zero_reading(PressureSensors::INHALATION_PIN);
-  sensorZeroVals[static_cast<int>(PressureSensors::EXHALATION_PIN)] =
-      get_raw_sensor_zero_reading(PressureSensors::EXHALATION_PIN);
+
+  auto set_zero_level = [](Sensor s) {
```

Comment:
Lambdas created in this way are guaranteed to not heap-allocate, right?

Reply:
Yeah, it's guaranteed to be identical to a "functor struct":

```
struct {
  int operator()(Sensor s) const { ... }
} set_zero_level;
```